.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/webblog
+    docker:
+      - image: circleci/python:3.6.4
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+      - run: python3 -m venv venv
+      - run:
+          command: |
+            . venv/bin/activate
```

Comment:
Why we need to activate the env every time?

Reply:
Every run command is independent in CircleCI. It is necessary to run this command if I want to use an installed module.

Reply:
I tried running it without activating again but it didn't work.